framework/src/consumption/lib/redshift-serverless/redshift-serverless-workgroup.ts

```diff
@@ -0,0 +1,279 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+import { createHash } from 'crypto';
+import { Names, RemovalPolicy, Stack } from 'aws-cdk-lib';
+import { Connections, IConnectable, IVpc, Port, SecurityGroup, SelectedSubnets } from 'aws-cdk-lib/aws-ec2';
+import { CfnConnection } from 'aws-cdk-lib/aws-glue';
+import { CfnWorkgroup } from 'aws-cdk-lib/aws-redshiftserverless';
+import { Construct } from 'constructs';
+import { RedshiftServerlessNamespace } from './redshift-serverless-namespace';
+import { RedshiftServerlessWorkgroupProps } from './redshift-serverless-workgroup-props';
+import { DataCatalogDatabase } from '../../../governance';
+import { Context, DataVpc, TrackedConstruct, TrackedConstructProps, Utils } from '../../../utils';
+import { RedshiftData } from '../redshift/redshift-data';
+
+const DEFAULT_PORT = 5439;
+
+/**
+ * Create a Redshift Serverless Workgroup. A default namespace would be created if none is provided.
+ * @see https://awslabs.github.io/data-solutions-framework-on-aws/docs/constructs/library/Consumption/redshift-serverless
+ *
+ * @example
+ * const workgroup = new dsf.consumption.RedshiftServerlessWorkgroup(this, "RedshiftWorkgroup", {
+ *    workgroupName: "redshift-workgroup"
+ * })
+ */
+export class RedshiftServerlessWorkgroup extends TrackedConstruct implements IConnectable {
+  private static readonly DEFAULT_VPC_CIDR = '10.0.0.0/16';
```

Comment:
No need of this, its the default CIDR in data VPC

Reply:
Can you make the `vpcCidr` optional in the `DataVpcProps`?